GOVERNANCE.rst

```diff
@@ -0,0 +1,143 @@
+TC members lead - and weigh in on - technical discussions and, if needed, can call for a vote of the TC for a technical decision.
+TC members can merge/close PRs and issues, and moderate (including block/mute) bad actors.
+The TC can propose governance changes to the SC.
+
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Discussion in the TC usually happens in the developer meetings. Developer meetings can be scheduled by any member of the SC or TC.
+
+If a member of the TC calls for a vote, the vote will be decided by the majority of the votest cast, provided that at least half of the TC members participate in the vote. If fewer than half of the TC members cast votes, the SC will make the decision according to the guidelines specified above.
+
+Votes are cast in a non-confidential manner.
+Decisions are documented in the developer meeting notes and/or in the GitHub repository.
+
+Individual TC members can suggest the addition of new contributors. The suggestion must be seconded by another TC member. Any TC member has the right to oppose the suggestion and call for a vote on the decision.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+TC members are the maintainers of AMReX.
+TC members are expected to attend and contribute to regular developer meetings.
+
+New TC members can be suggested by either SC or TC members. Confirmation of a new TC member requires a majority of votes cast by either the SC or the TC. The SC can veto any new appointment.
+
+Steering committee members can also be TC members.
+
+TC members can resign or be removed by majority vote by either TC or SC due to inactivity, inappropriate or otherwise negative behavior, or other reasons.
+
+
+Contributors
+------------
+
+Current Roster
+^^^^^^^^^^^^^^
+
+See: `GitHub contributors <https://github.com/AMReX-Codes/amrex/graphs/contributors>`__
+
+Role
+^^^^
+
+Contributors are valuable, vetted developers of AMReX.
+Contributions can be in many forms and not all need to be code contributions.
+Examples include code pull requests, support in issues & user discussions, writing and updating documentation, writing tutorials, visualizations, R&D on algorithms, testing and benchmarking, etc.
+Contributors can participate in developer meetings and weigh in on discussions.
+Contributors can "triage" (i.e. add labels to) pull requests, issues, and GitHub discussion pages.
+Contributors can comment on and review PRs (but not merge).
+
+Decision Process
+^^^^^^^^^^^^^^^^
+
+Contributors can individually decide on classification (triage) of pull requests, issues, and GitHub discussion pages.
+
+Appointment Process
+^^^^^^^^^^^^^^^^^^^
+
+Appointed after contributing to AMReX (see above) through nomination by any member of the TC. Another member of the TC must second the nomination.
+
+The role can be lost by resigning or by decision of an individual TC or SC member due to inactivity, inappropriate or otherwise negative behavior, or other reasons.
```

Comment:
```suggestion
The role can be lost by resigning or by majority vote of either the TC or the SC due to inactivity, inappropriate or otherwise negative behavior, or other reasons.
```